.gitignore

```
@@ -15,4 +15,5 @@ tests/.cache
 .vscode/
 .eggs/
 build/
-.python-version
\ No newline at end of file
+.python-version
+venv/
```

I know you use `venv/` only but many people may use `.venv`. Can you add the following ones as well?
```suggestion
venv/
.venv/
.env/
```

---

That sounds right to me.